core/services/workflows/models_test.go

```diff
@@ -8,6 +8,7 @@ import (
 )
 
 func TestParse_Graph(t *testing.T) {
+	t.Parallel()
```

Comment:
If the intent is to run subtests in parallel, would it not make sense to run Parallel method on the inner testing.T variable "st".

Reply:
I think it depends on the particular test. For fast sub-tests, it is possible to over-parallelize, and end up running longer overall from the extra overhead. In this case, the sub-test body is actually pure computation (as opposed to other places where we do integration style tests that are waiting for background services to produce some result) so it seemed unnecessary to parallelize the sub-tests. We could try it though :shrug:

Reply:
Ok, I just thought that there might be "a style guide" that would explicitly recommend against/for this option. Overall, I don't think that polluting the code with excessive Parallel calls seem useful; so, using them only on the top test level looks reasonable. Thanks